backup_and_restore/application_backup_and_restore/troubleshooting/oadp-backup-restore-cr-issues-doc.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-backup-restore-cr-issues-doc"]
+= OADP Backup and Restore CR issues
+include::_attributes/common-attributes.adoc[]
+:context: oadp-troubleshooting
+
+You might encounter these common issues with `Backup` and `Restore` custom resources (CRs).
+
+[id="backup-cannot-retrieve-volume-doc"]
+== Backup CR cannot retrieve volume
+
+The `Backup` CR displays the error message, `InvalidVolume.NotFound: The volume ‘vol-xxxx’ does not exist`.
+
+.Cause
+
+The persistent volume (PV) and the snapshot locations are in different regions.
+
+.Solution
+
+. Edit the value of the `spec.snapshotLocations.velero.config.region` key in the `DataProtectionApplication` manifest so that the snapshot location is in the same region as the PV.
+. Create a new `Backup` CR.
+
+[id="backup-cr-remains-in-progress-doc"]
+== Backup CR status remains in progress
+
+The status of a `Backup` CR remains in the `InProgress` phase and does not complete.
+
+.Cause
+
+If a backup is interrupted, it cannot be resumed.
+
+.Solution
+
+. Retrieve the details of the `Backup` CR:
++
+[source,terminal]
+----
+$ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
+  backup describe <backup>
+----
+
+. Delete the `Backup` CR:
++
+[source,terminal]
+----
+$ oc delete backup <backup> -n openshift-adp
```

Comment:
oc delete backup <backup> -n openshift-adp ==> oc delete backup <backup> -n {namespace}
So it will be in sync with step 1 and be more generic.